electron/gog/setup.ts

```diff
@@ -41,8 +41,9 @@ async function setup(appName: string): Promise<void> {
   const crossoverEnv =
     isCrossover && crossoverBottle ? `CX_BOTTLE=${crossoverBottle}` : ''
   const isProton =
-    gameSettings.wineVersion.name.includes('Proton') ||
-    gameSettings.wineVersion.name.includes('Steam')
+    (gameSettings.wineVersion.name.includes('Proton') ||
+      gameSettings.wineVersion.name.includes('Steam')) &&
+    !gameSettings.wineVersion.name.includes('lutris')
```

Comment:
The whole approach to differentiating between Proton, Wine and CX by checking the name/path is hacky.  
Instead of adding onto those hacks, we should modify the frontend to now respect `WineInstallation.type`, which will almost always be correct (one edge case exists when adding custom Wine versions, I'm planning on fixing that up soon)

Reply:
It looks like using `WineInstallation.type` in `electron/launcher.ts` and `electron/gog/setup.ts` works, but the `WineInstallation` in `src/types.sh` doesn't have `type`.

Reply:
> but the `WineInstallation` in `src/types.ts` doesn't have type.

This is just caused by me forgetting to update that, you can replace the definition with the one from `electron/types.ts`